lib/plugin.js

```diff
@@ -1,6 +1,8 @@
 /* eslint-disable import/no-unresolved, import/no-extraneous-dependencies, prefer-template */
 import middleware from '@@/.nuxt/middleware'
 
+// ToDo: TTL hardcoded
+const TOKEN_TIME_TO_LIVE = 2 * 60 * 1000 // 2 minutes
```

Comment:
Where is this value coming from?

Reply:
Should probably come from identity service, for now token lives 2min, usually should be 2h